express-notes-api/index.js

```diff
@@ -0,0 +1,114 @@
+// eslint-disable-next-line no-console
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+const jsonData = require('./data.json');
+const notes = Object.values(jsonData.notes);
```

Comment:
I recommend creating this array _inside_ your `app.get('/api/notes', middleware)`. This array will only hold the notes that were in the file when the server started, unless your other routes keep it up-to-date.